### ISSUE TITLE
getConflictingBundle throws mismatch EIP-1559 gasPrice != maxFeePerGas

### ISSUE THREAD

**Issue by j05u3** [flashbots/ethers-provider-flashbots-bundle#70]
It happens with getConflictingBundle processes the first transaction of target block 14635131 (this one: https://etherscan.io/tx/0x7450798650f46ba9faf09a57d3a1e999763aba2aa9bb00b9f35e07216b6eeea7). Details:

```
Error: mismatch EIP-1559 gasPrice != maxFeePerGas (argument="tx", value={"gasPrice":{"type":"BigNumber","hex":"0x02ab9f29f6"},"maxFeePerGas":{"type":"BigNumber","hex":"0x046adcee56"}}, code=INVALID_ARGUMENT, version=transactions/5.6.0)     
    at Logger.makeError (C:\repos\blockchain\simple-arbitrage\node_modules\@ethersproject\logger\src.ts\index.ts:261:28)    at Logger.throwError (C:\repos\blockchain\simple-arbitrage\node_modules\@ethersproject\logger\src.ts\index.ts:273:20)
    at Logger.throwArgumentError (C:\repos\blockchain\simple-arbitrage\node_modules\@ethersproject\logger\src.ts\index.ts:277:21)
    at _serializeEip1559 (C:\repos\blockchain\simple-arbitrage\node_modules\@ethersproject\transactions\src.ts\index.ts:174:20)
    at serialize (C:\repos\blockchain\simple-arbitrage\node_modules\@ethersproject\transactions\src.ts\index.ts:319:20) 
    at C:\repos\blockchain\simple-arbitrage\src\FlashbotsConflictingBundle.spec.ts:163:38
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Promise.all (index 0)
    at async main (C:\repos\blockchain\simple-arbitrage\src\FlashbotsConflictingBundle.spec.ts:156:36) {
  reason: 'mismatch EIP-1559 gasPrice != maxFeePerGas',
  code: 'INVALID_ARGUMENT',
  argument: 'tx',
  value: {
    gasPrice: BigNumber { _hex: '0x02ab9f29f6', _isBigNumber: true },
    maxFeePerGas: BigNumber { _hex: '0x046adcee56', _isBigNumber: true }
  }
}
```

**Comment by j05u3** [comment]
Just submitted a PR fixing it (tested in my computer): https://github.com/flashbots/ethers-provider-flashbots-bundle/pull/71

**Comment by epheph** [comment]
Released v0.4.4, thanks
